src/index.html

```diff
@@ -12,6 +12,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0">
     <meta name="theme-color" content="#ffb400">
     <title>Vanilla Hacker News PWA</title>
+    <link rel="stylesheet" href="style.css">
```

Comment:
This is a great idea and usually it makes it all together better to work with! In this case though, my goal is to minimize as much as possible external loading resources, so the browser would load and parse everything together.

Reply:
If you want I can do it in the same html.

Reply:
That would be awesome! Thank you!